.rubocop.yml

```diff
@@ -0,0 +1,60 @@
+AllCops:
```

Comment:
Can you put in a comment as a first line, where did you find this configuration ?

Reply:
I worked for a US company where we used this configuration and it works well, but on PR, not push what I find unnecessary.
For tests, I wait for the app to grow and to see what tests we use. Btw I never used more than the configuration  I made.
Repo with linter is private and I can't show the source, sorry.

Reply:
Ok, it worked both locally and for GitHub. With the correction of errors easily locally, GitHub was to pass the linter test.
Now I will delete the branch and then Pr will be on the new one.